### ISSUE TITLE
Endnotes/sidenotes

### ISSUE THREAD

**Issue by jasloe** [dramonline/soundamerican#20]
This issue tracks code and discussion on an endnote/sidenote implementation.

**Comment by natewooley** [comment]
I think for now, endnotes will be easier and will echo the print version.

**Comment by jasloe** [comment]
Agreed. I understand the urgency to get this out the door. 

However, going forward and as an iteration of the product, sidenotes are visually attractive as well as quite practical.

**Comment by natewooley** [comment]
Yes, I totally understand that. Do you think that articles featuring a lot of academic attribution will be good candidates for sidenotes as well, or just pieces in which the note elucidates something in the text...or maybe it really doesn't matter either way.

**Comment by jasloe** [comment]
It's more of a user interface question than anything, I suppose. Consecutive very long citations in a side note may be problematic, something which the Tufte guidelines don't account for.
